examples/default.yaml

```diff
@@ -39,9 +39,11 @@ images:
 - location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240821/ubuntu-24.04-server-cloudimg-amd64.img"
   arch: "x86_64"
   digest: "sha256:0e25ca6ee9f08ec5d4f9910054b66ae7163c6152e81a3e67689d89bd6e4dfa69"
+  cid: "bafybeievi2i673t7kgzx6vsxc6lod3bvagzc364e6kes43p6catfowgone"
```

Comment:
I don't think we should add the cid to the built-in templates unless the distro publishes the cid by themselves, as computing the cid incurs downloading and reading the whole images.

The same applies to contained too.

You may still add experimental/ubuntu-ipfs.yaml with `location: ipfs://CID`.
I still think the locator object doesn't need a dedicated field for `cid`

Reply:
Also hack/update-template.sh should skip ubuntu-ipfs.yaml by default

Reply:
I can remove the CID feature, than the IPFS would be more similar to the ORAS.